contracts/staking/PoSValidatorManager.sol

```diff
@@ -312,10 +301,27 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         });
     }
 
-    function resendEndDelegation(bytes32 delegationID) external {
-        _checkPendingEndDelegatorMessage(delegationID);
+    function resendUpdateDelegation(bytes32 delegationID) external {
```

Comment:
Can we add a comment here explaining why it's safe to always resend the latest nonce and weight even it it's not the nonce and weight that were originally sent for the given message?